ext/session/session.stub.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/** @return string|false */
+function session_name(string $name) {}
+
+/** @return string|false */
+function session_module_name(string $module) {}
+
+/** @return string|false */
+function session_save_path(string $path) {}
+
+/** @return string|false */
+function session_id(string $id) {}
+
+/** @return string|false */
+function session_create_id(string $prefix) {}
+
+function session_regenerate_id(bool $delete_old_session): bool {}
+
+function session_decode(string $data): bool {}
+
+/** @return string|false */
+function session_encode() {}
+
+function session_destroy(): bool {}
+
+function session_unset(): bool {}
+
+/** @return int|false */
+function session_gc() {}
+
+function session_get_cookie_params(): array {}
+
+function session_write_close(): bool {}
+
+function session_abort(): bool {}
+
+function session_reset(): bool {}
+
+function session_status(): int {}
+
+function session_register_shutdown(): void {}
+
+function session_commit(): bool {}
+
+function session_set_save_handler($open, $close, $read, $write, $destroy, $gc, $create_sid = null, $validate_sid = null, $update_timestamp = null): bool {}
```

Comment:
Due to the overloaded signature, all but the first parameter should be optional.